src/System Application/App/AppSource Gallery/src/AppSourceProductManager.Codeunit.al

```diff
@@ -576,26 +500,9 @@ codeunit 2515 "AppSource Product Manager" implements "IAppSource Product Manager
 
     local procedure SetDefaultDependencyImplementation()
     var
-        dependencyInstance: Codeunit "AppSource Product Manager";
+        DependencyInstance: Codeunit "AppSource Product Manager";
```

Comment:
```suggestion
        AppSourceProductManager: Codeunit "AppSource Product Manager";
```

if you're unhappy with the names of your variables, change the name of the object...